ModularBungalow/meele_weapons.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/melee/baseball_bat/toolbox
+	name = "Baseball bat with a toolbox"
+	desc = "There ain't a skull in the league that can withstand a toolbox."
+	icon = 'ModularBungalow/zbungalowicons/weapons/melee.dmi'
+	icon_state = "baseball_toolbox"
+	inhand_icon_state = "baseball_toolbox"
+	lefthand_file = 'ModularBungalow/zbungalowicons/weapons/melee_lefthand.dmi'
+	righthand_file = 'ModularBungalow/zbungalowicons/weapons/melee_righthand.dmi'
+	force = 11
+	wound_bonus = -5
+	throwforce = 12
```

Comment:
you don't need this, parent has it

Reply:
different stats, except the throw force, that i will delete. thanks anyway for the other stuff. luv u